File: repos_estudiantes_TARQUI_tercera_entrega/ILLANES_ESCOBAR_FERNANDO_JOHSUA/PERSISTENCIA/Ejercicio5/Java/Main.java
package PERSISTENCIA.Ejercicio5.Java;

public class Main {
    public static void main(String[] args) {
        GestorFarmacia gestor = new GestorFarmacia();

        Sucursal s1 = new Sucursal(1, "Av. Villazón #123");
        s1.agregarMedicamento(new Medicamento("Golpex", "tos"));
        s1.agregarMedicamento(new Medicamento("Ibuprofeno", "dolor"));

        Sucursal s2 = new Sucursal(2, "Calle Bueno #456");
        s2.agregarMedicamento(new Medicamento("Vitamina C", "inmunidad"));

        gestor.guardarSucursal(s1);
        gestor.guardarSucursal(s2);

        gestor.mostrarMedicamentosTos(1);
        gestor.mostrarSucursalesConGolpex();
    }
}

File: repos_estudiantes_TARQUI_tercera_entrega/ILLANES_ESCOBAR_FERNANDO_JOHSUA/PERSISTENCIA/Ejercicio5/Python/main.py
from medicamento import Medicamento
from sucursal import Sucursal
from gestor_farmacia import GestorFarmacia

gestor = GestorFarmacia()

s1 = Sucursal(1, "Av. Villazón #123")
s1.agregar_medicamento(Medicamento("Golpex", "tos"))
s1.agregar_medicamento(Medicamento("Paracetamol", "dolor"))

s2 = Sucursal(2, "Calle Bueno #456")
s2.agregar_medicamento(Medicamento("Vitamina C", "inmunidad"))

gestor.guardar_sucursal(s1)
gestor.guardar_sucursal(s2)

gestor.mostrar_medicamentos_tos(1)
gestor.mostrar_sucursales_con_golpex()
